maps/away/away_site/tajara/saniorios_outpost/saniorios_outpost_ghostroles.dm

```diff
@@ -0,0 +1,36 @@
+/datum/ghostspawner/human/saniorios_outpost_crew
+	short_name = "saniorios_outpost_crew"
+	name = "Sani'Orios Asteroid Outpost Crewmember"
+	desc = "Crew a Democratic People's Republic of Adhomai's asteroid outpost in Sani'Orios."
```

Comment:
desc should explain more about this role - like what do they do?
there should also be welcome_text or whatever, that shows on spawn, and explains things more in detail